ClientLibrary/lib/build.gradle

```diff
@@ -2,19 +2,25 @@ apply plugin: 'com.android.library'
 
 android {
     compileSdkVersion 22
-    buildToolsVersion '27.0.3'
 
     defaultConfig {
         minSdkVersion 22
         targetSdkVersion 22
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
 }
 
 dependencies {
-    implementation 'com.google.code.gson:gson:2.8.2'
+    implementation 'com.google.code.gson:gson:2.8.5'
+    implementation 'com.squareup.okhttp3:okhttp:3.13.1'
 }
 
+
```

Comment:
[Nit] Remove this.

Reply:
Yes, that shouldn't have reached the pull request